src/senfd/documents/enriched.py

```diff
@@ -473,36 +473,34 @@ class CommandIoOpcodeFigure(EnrichedFigure):
     command_set_name: str
 
 
-class GeneralCommandStatusValueFigure(EnrichedFigure):
-    REGEX_FIGURE_DESCRIPTION: ClassVar[str] = r".*General.Command.Status.Values.*"
+class StatusCodeFigure(EnrichedFigure):
+    REGEX_FIGURE_DESCRIPTION: ClassVar[str] = r"^(Status.Code.-).*(Type|Error).Values$"
     REGEX_GRID: ClassVar[List[Tuple]] = [
         REGEX_GRID_VALUE,
```

Comment:
We actually want seperate classes for these as to model these as Generic and Specific Values.
However, it might be that this should be "enriched" later on when producing the "merged model-document". So, let's keep the nice simplification of the codebase which this is.